src/GDGUkraine/templates/card.html

```diff
@@ -29,7 +30,56 @@
         div {
             text-align: justify;
         }
+        .approve-btn:hover {
+            background-color: black; 
+            color: white;
+        }
+        .btn {
+                        border-radius: 5px;
+            display: block;
+            padding: 10px 5px;
+            margin: 0 auto;
+            cursor: pointer;
+            width: 100px;
+            border: 0;
+            box-shadow: none;
+        }
+        .approve-btn {
+
+            color: black;
+            background-color: rgb(240,240,240);
+        }
+        .btn-success {
+    color: #fff;
+    background-color: #5cb85c;
+
+    border-color: #4cae4c;
+}
     </style>
+    <script type="text/javascript">
+        function approve_visit () {
+            $.ajax({
+              url: "http://localhost:8080/api/events/{{registration.id}}/check-in",
+              type: "POST",
+              data:'{}',
+              contentType:"application/json; charset=utf-8",
+                dataType: 'json',
+             
+            }).done(function( data, status) {
+                        if (data['visited']) {
+                            $('.approve-btn').html('Successfully approved');
+                            $('.approve-btn').attr('disabled','disabled');
+                            $('.approve-btn').addClass('btn-success ');
+                            $('.approve-btn').removeClass('approve-btn');
+                        } else {
+                            console.log(data);
+                            $('.result').html('Some error happends');
```

Comment:
> happened.

It's better to be more specific. I would write smth like `Couldn't perform check-in for this person`.